NewRelic/NewRelicInteractor.php

```diff
@@ -127,4 +127,29 @@ public function setUserAttributes(string $userValue, string $accountValue, strin
     {
         return newrelic_set_user_attributes($userValue, $accountValue, $productValue);
     }
+
+    public function getTraceMetadata(): array
+    {
+        return newrelic_get_trace_metadata();
```

Comment:
IMHO we should add a assertion in case the user installed the wrong version of the extension.

```php
if (!function_exists('newrelic_get_trace_metadata')) {
  throw the BadMethodCallException('You need the "newrelic" extension version ??? or higher to use this method');
}